markwon/src/main/java/ru/noties/markwon/renderer/SpannableBuilder.java

```diff
@@ -0,0 +1,63 @@
+package ru.noties.markwon.renderer;
+
+import android.support.annotation.Nullable;
+import android.text.SpannableStringBuilder;
+
+/**
+ * Copied with modification and renaming from @see <a href="https://github.com/Uncodin/bypass/blob/master/platform/android/
+ * library/src/in/uncod/android/bypass/ReverseSpannableStringBuilder.java">Uncodin/bypass</a>
+ */
+public class SpannableBuilder extends SpannableStringBuilder {
```

Comment:
Let's maybe keep it in original package? I understand it fit here most, but let's make this internal change that would not require users to do changes. It's minor issue, but still

Reply:
Will move it...